src/docs/product/issues/states-triage/index.mdx

```diff
@@ -1,80 +1,60 @@
+or
 
-<Note>
-
-Metric alerts can be triggered by ignored issues.
-
-</Note>
+- By [including the issue ID in a commit](/product/releases/associate-commits/?original_referrer=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fissues%2Fstates-triage%2F#resolve-issues-by-commit)
 
-### Mark Reviewed
-
-Mark an issue as reviewed when you want to acknowledge that you've seen the issue but don't want to ignore it; that is, you intend to fix it at some point in the future. Marking an issue reviewed removes the reason label and removes it from the **Review List** in the "For Review" tab. The "Mark Reviewed" action is only available on issues flagged for review.
-
-While you can work to have zero unresolved issues ("Inbox Zero") by resolving or ignoring all issues, this usually isn't feasible for any non-trivial production application. However, it is feasible to have zero for-review issues by triaging the issues in the **Review List**.
-**We recommend triaging these issues once a day to keep your issues manageable.**
-
-### Assign
-
-Assign issues to teams or individuals to route them to the right people and help you and your team see only relevant issues. Typically, assigning an issue to a team indicates that the team owns the issue and that they're responsible for triaging it further. In some cases, you can assign issues manually or automatically using [ownership rules and code owners](/product/issues/ownership-rules/). For error issues, on the issue list, you might also see a list of suggested assignees including the owners of suspect commits.
-
-### Link Issues
-
-Create an issue in an external project management tool like Jira that tracks the Sentry issue, or link an existing one. [Two-way sync](/product/accounts/getting-started/#33-issue-tracking) is available for some tools. Issues linked using [public integrations](/product/integrations/integration-platform/public-integration) or [internal integrations](/product/integrations/integration-platform/internal-integration) are searchable using the terms `is:linked` and `is:unlinked`. You can also set up Sentry issues to be automatically linked to tickets (or issues) in certain integrations using [alert actions](/product/alerts-notifications/issue-alerts/).
+If the same Issue comes back, its status will be automatically changed to `Regressed`.
 
 ### Delete
 
-Delete an issue to remove it from [sentry.io](https://sentry.io). A new issue is created if a new instance of the issue (that is, a new event) occurs.
+You can remove an Issue from the Issue list by deleting it, but it will reappear as a new Issue if it reoccurs.
 
-<Include name="only-error-issues-note.mdx" />
+There's also an option to `Delete and Discard Forever`, which will make it so that the Issue is never seen again, even if it reoccurs. Any future events will be discarded automatically and won't count towards [your quota](/product/accounts/quotas/).
```

Comment:
```suggestion
There's also an option to `Delete and Discard Forever`, which will make it so that the Issue is never seen again, even if it recurs. Any future events tied to the permanently deleted Issue will be discarded automatically and won't count towards [your quota](/product/accounts/quotas/).
```